sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/ArmApiManagementModelFactory.cs

```diff
@@ -0,0 +1,385 @@
+        /// <param name="license"> License information for the API. </param>
+        /// <param name="sourceApiId"> API identifier of the source API. </param>
+        /// <param name="displayName"> API name. Must be 1 to 300 characters long. </param>
+        /// <param name="serviceUri"> Absolute URL of the backend service implementing this API. Cannot be more than 2000 characters long. </param>
+        /// <param name="path"> Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance. It is appended to the API endpoint base URL specified during the service instance creation to form a public URL for this API. </param>
+        /// <param name="protocols"> Describes on which protocols the operations in this API can be invoked. </param>
+        /// <param name="apiVersionSet"> Version set details. </param>
+        /// <returns> A new <see cref="ApiManagement.ApiData"/> instance for mocking. </returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static ApiData ApiData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, string description, AuthenticationSettingsContract authenticationSettings, SubscriptionKeyParameterNamesContract subscriptionKeyParameterNames, ApiType? apiType, string apiRevision, string apiVersion, bool? isCurrent, bool? isOnline, string apiRevisionDescription, string apiVersionDescription, ResourceIdentifier apiVersionSetId, bool? isSubscriptionRequired, Uri termsOfServiceUri, ApiContactInformation contact, ApiLicenseInformation license, ResourceIdentifier sourceApiId, string displayName, Uri serviceUri, string path = null, IEnumerable<ApiOperationInvokableProtocol> protocols = null, ApiVersionSetContractDetails apiVersionSet = null)
+        {
+            return ApiData(
+                id: id,
+                name: name,
+                resourceType: resourceType,
+                systemData: systemData,
+                description: description,
+                authenticationSettings: authenticationSettings,
+                subscriptionKeyParameterNames: subscriptionKeyParameterNames,
+                apiType: apiType,
+                apiRevision: apiRevision,
+                apiVersion: apiVersion,
+                isCurrent: isCurrent,
+                isOnline: isOnline,
+                apiRevisionDescription: apiRevisionDescription,
+                apiVersionDescription: apiVersionDescription,
+                apiVersionSetId: apiVersionSetId,
+                isSubscriptionRequired: isSubscriptionRequired,
+                termsOfServiceUri: termsOfServiceUri,
+                contact: contact,
+                license: license,
+                sourceApiId: sourceApiId,
+                displayName: displayName,
+                serviceUri: serviceUri,
+                path: path,
+                protocols: protocols?.ToList(),
```

Comment:
well we do not need to call `ToList` here because the generated version of this method will call it anyway. Now we are calling `ToList` twice.
```suggestion
                protocols: protocols,
```